app/assets/stylesheets/shared/header.scss

```diff
@@ -33,53 +33,50 @@
 }
 
 .header-logo {
-  float: left;
+  @include transition(1.0s);
+
+  background-size: cover;
```

Comment:
Properties should be ordered background, background-size, display, float, height, margin, width